vanilla/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -8,7 +8,6 @@
 
```

Comment:
Any reason why this file had to be changed ? Seems to be outside the `mobx` folder

Reply:
Rest looks good, so I can merge once we remove this file from the changelist

Reply:
This file is changed because I ran the Vanilla the implementation project on an iOS simulator to just see how things looks. So this is an automatic change during the build for Xcode.

Reply:
Also still there are unit and integration tests left to write as mentioned in https://github.com/brianegan/flutter_architecture_samples/blob/master/app_spec.md

Reply:
Ya, we can do that in a separate PR. Let's get all the functionality in and match the spec. We can then add tests to match it

Reply:
So should I do something about that Xcode file that has changed?

Reply:
May be just revert the change

Reply:
That will be a bit tricky but I will do that by EOD.